### ISSUE TITLE
Accessibility bug: Loss of keyboard focus is found after the editor section

### ISSUE THREAD

**Issue by LadyNaggaga** [MicrosoftDocs/feedback#2531]
Video for the bug [Loss of keyboard focus is found after the editor section](https://github.com/dotnet/try/files/4207892/789_A11y_.NETOnlineREPL_.NET.Editor_Keyboard.pptx)

**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

#A11yTCS;#A11ySev2;#A11y_.NETOnlineREPL;#.NET Online REPL;#Benchmark;#DesktopWeb;#Win10;#WCAG2.4.7;#Accessibility;#.NET Online REPL;#keyboard;#ChromiumEdge;

​**Repro Steps:​**
1.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
2.Tab till "Enter focus Mode" button and press enter.
3.Navigate using the tab key in the editor section.
4.Observe the keyboard focus after the editor section. 

**Actual Result:​**
When navigated using the tab key after the editor there is loss of keyboard focus is observed.

**Expected Result:​**
There should be no loss of keyboard focus after the editor section.

**User Impact:​**
Keyboard and low vision users will find difficulty in identifying the keyboard focus after the editor section.

**Attachment:**
[789_A11y_.NETOnlineREPL_.NET Editor_Keyboard.pptx](https://github.com/dotnet/try/files/4207892/789_A11y_.NETOnlineREPL_.NET.Editor_Keyboard.pptx)
